config/opa-service/rego/context_broker_policy.rego

```diff
@@ -31,6 +33,18 @@ path_matches_policy(entry, request) {
   current_path[1] == "v2"
   current_path[2] == "entities"
   current_path[3] == entry.resource
+  not entry.constraint
+}
+
+path_matches_policy(entry, request) {
+  entry.resource_type == "entity"
+  current_path := split(request.resource, "/")
+  current_path[1] == "v2"
+  current_path[2] == "entities"
+  current_path[3] == entry.resource
+  constraints := split(entry.constraint, " ")
+  constraints[0] == "acl-oc:ResourceName"
```

Comment:
how would we use this in teadal? every API would need to have its own rego file w/ a `path_matches_policy` definition I suppose. That's fair enough, but then we'd have to duplicate the constraints logic in each and every file? also, what if I want to check request fields (e.g. method) or data fields in the JSON payload in the case of a POST?